### DISCUSSION TITLE
M64plus Fzpro S22 ultra emulator setup

### DISCUSSION THREAD

**Discussion by KenDinelly2057** [mupen64plus-ae/mupen64plus-ae#1151] [Q&A]
Hello, I would like to get technical support for the m64plus Fzpro emulator.  I am unable to run the Nba hangtime game on the n64.  It is having problems in the emulation such as: fps drops, audio delays. I am using this Pro version on my Smartphone S22 Ultra 12 gb of RAM and 256 gb of Rom, Android13. Before, it ran normally!

**Comment by fzurita** [comment]
Nothing had changed in the app in a long time, so it's more likely something that changed in your phone or a setting was modified.

A possibility is that the selected emulation profile was changed. You can try different emulation profiles by selecting the game, going to settings, and choosing a different emulation profile there.
